### PR TITLE
Tamil Language support

### PR DESCRIPTION
**Pull request by abuabdul**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes, Indian Tamil Language Translation
| BC breaks?      | no
| Related tickets | no
| License         | Apache License v2.0

**Demo:**
https://b2oplus.github.io/bootstrap-datepicker/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ docs/_build
 # OS or Editor folders
 .DS_Store
 .idea
+*.iml
```
Comment (acrobat) [inline .gitignore]: Can you revert this line?

**Comment by acrobat** [comment]
Hi @abuabdul, Thanks for the translation only one comment left to fix!

File: .gitignore (hunk 1 above)
Comment (abuabdul) [inline .gitignore]: Sure. Will do.

**Comment by abuabdul** [comment]
@acrobat I have fixed it.

**Comment by acrobat** [comment]
Thanks @abuabdul

**Comment by Azaret** [comment]
I believe that at the moment we enforce usage of ISO 639-1 unless we want to add a variant or a language for a specific country.  
I won't speculate for the usage of Tamil, but:  

- if there is officially only one Tamil dialect
- or if there is no significant differences between any Tamil dialects in general or in the particular context of this date picker (rewording: if the result of various Tamil dialects of this translated file result the same) 
- or if the Indian Tamil is largely the most used of the Tamil dialects, 

Then we should rename the file and internal name to `ta`.

**Comment by acrobat** [comment]
Thanks @Azaret for the feedback! I've renamed the locale file in 9f5e9a5

**Comment by Azaret** [comment]
Well I would have a word from @abuabdul on this 😄

**Comment by acrobat** [comment]
Yeah I was a bit quick with the changes. I'm going to revert my commit for now and wait for the feedback from @abuabdul. If the locale key needs to change we can do it in a 1.7.x release!

**Comment by abuabdul** [comment]
Hi, I guess @Azaret is right. The translation will be mostly same if other dialects like Malaysian Tamil or Colombo Tamil did translate. The dialects of Tamil could be different regionally but the calendar translation is same since it is a Gregorian. So i will agree to rename. Thanks @Azaret.

@acrobat let me know if you want me to rename the file.

**Comment by acrobat** [comment]
@abuabdul Yeah if you also agree that the locale code should be 'TA' then please create a PR to rename the file and locale code in the file. Thanks!

**Comment by Azaret** [comment]
What a mess @acrobat 😋  
I'm gonna PR this.  
Ty for your help @abuabdul !

**Comment by acrobat** [comment]
Yeah, I'm sorry :D Thanks @Azaret and @abuabdul

**Pull request merged**